Xamarin.Forms.Platform.iOS/Renderers/ShellFlyoutContentRenderer.cs

```diff
@@ -273,6 +283,54 @@ public override void ViewDidLoad()
 			UpdateFlowDirection();
 		}
 
+		void UpdateFlyoutContent()
+		{
+			var view = (_shellContext.Shell as IShellController).FlyoutContent;
+
+			if (_flyoutContent != null && _flyoutContent != view)
+			{
+				var oldRenderer = Platform.GetRenderer(_flyoutContent);
+				var oldContentView = _flyoutContentView;
+
+				_flyoutContent.ClearValue(Platform.RendererProperty);
+				oldContentView?.RemoveFromSuperview();
+
+				_shellFlyoutContentManager.Content = null;
+				_shellFlyoutContentManager.ContentView = null;
+				_flyoutContent = null;
+
+				oldRenderer?.Dispose();
+			}
+
+			if (view == null)
+			{
+				View.InsertSubview(_tableViewController.View, 0);
+				_shellFlyoutContentManager.ContentView = _tableViewController.TableView;
+				_shellFlyoutContentManager.Content = null;
+				_shellFlyoutContentManager.LayoutParallax();
+				_shellFlyoutContentManager.SetHeaderContentInset();
+				return;
+			}
+
+			if (_flyoutContent == view)
+				return;
+
+			_tableViewController.View.RemoveFromSuperview();
+
+			_flyoutContent = view;
+
+			if (_flyoutContent != null)
```

Comment:
we don't need to check for null here, do we? we set the `_flyoutContent ` to the `view` value. If `view` is null we return the code at line 312, and we don't get this line.

Reply:
Removed this code